### DISCUSSION TITLE
order of the menu

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#366] [Q&A]
can i change the order?

![grafik](https://github.com/user-attachments/assets/f3c51c4b-bb48-4e58-b43f-7b094f0788a6)

i want download at first place

**Comment by kwal559** [comment]
Go to User, user settings, and adjust .."buttons under main images"

ButtonsUnderMainImages:
A list of what buttons to include directly under images in the main prompt area of the Generate tab.
Other buttons will be moved into the 'More' dropdown.
This should be a comma separated list.
The following options are available: "Use As Init", "Edit Image", "Upscale 2x", "Star", "Reuse Parameters", "Open In Folder", "Delete", "View In History", "Refine Image"
The default is blank, which currently implies 'Use As Init,Edit Image,Star,Reuse Parameters'

Also, there is an output folder..which can a be modified.
And another question you had, well - some models come with vae and encoders built in, or they can be universal, or swarm can auto pick for you.  And I agree about image results differ from each UI.  that's normal. It's even normal to not get the same image as others when using the same seed.  (i've never got mildly close when copy/paste civicAI configs). At least for me, every little thing adds randomness, from loras, to resolution, to spelling/grammar syntax. 
For presets question, if you have toggled on an option, like, a seed number, or a sampler, you can't change it until you remove the preset.  careful when you change somethings, the 'default' might display although it's using the preset internally. You may have better luck using the 'simple' generate option.

**Comment by kalle07** [comment]
thx,
but i see and i tryed ... "download" is not possible to put at first
thats i think the button all are most using ... okay for me ;)
one klick to save an imaga is a must, i think

and than delete and download such close, thats ???


for image results are different from any UI, which is not really satisfactory ^^ 
for civitai, iv one reason, for numbers that higer than (12 digits, i dont remember maybe more) civita automatically rounds up ....
but in my case everything is local with 10 digits and everything is the same as far as I can see

fo presets, i know i think every one need it in a different way, but in the cases I have described, it is somewhat strange and incomprehensible
and it give two ways, only choose and apply ... i talked from apply


but all in all swarm is okay , not that confusing like invoke and not that much erros like forge ;)
one is missing or i dont found, a simple inpaint option with drawing a mask and so one ...

**Comment by mcmonkey4eva** [comment]
fwiw if I need to download an image, I usually either Right Click -> Save Image as, or I just drag the image from the browser to my desktop. That's why "Download" button is default to under the "More" part, it's rare to actually need it.

Delete is hidden by default just so you don't do it accidentally.

re `a simple inpaint option with drawing a mask and so one`, click "Edit Image"

btw you're posting a ton on GitHub, are you aware there's a Discord? We use it for most things rather than GitHub, GitHub posts are usually just used to track feature requests and bugs. https://discord.gg/q2y38cqjNw

**Reply by kalle07** [reply]
i see may a feature , because one klick is the fastest

posts... yes i know but
discord is another data collector and extremely confusing design, i dont like it   ;)
but here is a discussion area too

**Reply by dlgptff** [reply]
Is there a way to detach the GENERATE button from the prompt window and move it to another location? On monitors with 4:3, this button takes up a lot of space, and in my case, for example, it would be more convenient to move it to the right panel.
